WinUIGallery/ControlPages/DesignGuidance/IconographyPage.xaml

```diff
@@ -206,6 +208,27 @@
                         <controls:SampleCodePresenter x:Name="XAMLCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
                         <TextBlock Text="C#" />
                         <controls:SampleCodePresenter x:Name="CSharpCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
+                        <TextBlock Text="Tags" />
```

Comment:
I think we need to start have those headers differentiate themselves a bit better from the content. Especially for the "No tags", I think its more difficult than necessary to know what the header is and what the actual content is.

Reply:
I can take that up in a separate PR!

Reply:
@niels9001 I'll leave this to you then.